source/edge-server/manage-edge-servers.txt

```diff
@@ -0,0 +1,485 @@
+.. _edge-server-manage-atlas:
+
+============================
+Manage Edge Servers in Atlas
+============================
+
+.. meta::
+   :description: Learn how to create an Edge Server Service, and create, delete, or check the status of Edge Servers.
+
+.. facet::
+   :name: genre 
+   :values: tutorial
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Before You Begin
+----------------
+
+.. tabs-realm-admin-interfaces::
+
+   .. tab::
+      :tabid: ui
+
+      You need the following to manage Edge Servers in the Atlas UI:
+
+      - .. include:: /includes/prereqs/atlas-account.rst
+
+   .. tab::
+      :tabid: api
+
+      You need the following to manage Edge Servers with the Admin API:
+
+      - .. include:: /includes/prereqs/atlas-account.rst
+
+      - .. include:: /includes/prereqs/atlas-admin-api-key.rst
+
+      - .. include:: /includes/prereqs/api-project-app-id.rst
+
+.. _create-edge-server-service:
+
+Create an Edge Server Service
+-----------------------------
+
+The Edge Server Service is your Atlas destination to create, delete, or check
+on your Edge Server instances. 
+
+There are two types of services that enable you to use Edge Server:
+
+- Edge Server Service: use this option if you want clients to connect to the 
+  Edge Server with MongoDB Drivers and tools. It infers and updates the schema 
+  from the documents you insert, and provides a developer experience familiar 
+  to MongoDB developers. The Edge Server Service rejects incoming Atlas Device
+  SDK connections.
+- Device Sync Service, adding Edge Servers: use this option if you want to 
+  develop client apps using Atlas Device SDK. This option involves more strict 
+  schema enforcement, which the SDKs use to automatically sync objects 
+  across devices and handle conflict resolution.
+
+Regardless of which type of service you choose, Edge Servers use Device Sync 
+to sync data between the Edge Server instance and Atlas. Even when you choose 
+an Edge Server Service, you must first enable Device Sync before you can create 
+Edge Server instances.
+
+.. important:: You can't add Atlas Device SDK Apps later to an Edge Server Service
+
+   You can always add Edge Servers to a Device Sync Service. But you can't use
+   Atlas Device SDK connections with an Edge Server Service. 
+   
+   Users are not portable between services. If you start with an Edge Server 
+   Service, but later want to add an Atlas Device SDK application, there is no 
+   way to transfer any users in your Edge Service to a new Device Sync Service.
+
+You can create an Edge Server-compatible service in Data Services or App 
+Services. 
+
+- You might select Data Services when you just want to get started building
+  with Edge Server or Device Sync and Edge Server, and don't need other 
```

Comment:
Do we want this to say "with an Edge Server" or "with Edge Servers"? I guess it makes sense to maybe keep this as is since Edge Server is the product name but was just a question.

Reply:
Yeah, I went with "Edge Server" as the product name here, but am willing to change it if it seems confusing.

Reply:
Maybe good to get Ian's input here, I don't know the right answer. I can raise this to him in the channel.